layers/meta-balena-raspberrypi/recipes-kernel/linux-kunbus/linux-kunbus.bb

```diff
@@ -1,8 +1,8 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-LINUX_VERSION = "5.10.120"
+LINUX_VERSION = "5.10.152"
 
-SRCREV = "raspberrypi-kernel_1%9.20220728-5.10.120+revpi1"
```

Comment:
Can you add these changes in the first commit please? Here I mean https://github.com/balena-os/balena-raspberrypi/pull/917/commits/ba0e2bc3a7e1f6f7d99741cdc39ac24da0709782
So we don't have multiple commits that change twice the kernel revision and keeps the git history cleaner